docs/adr/0009-cluster-resources.md

```diff
@@ -0,0 +1,95 @@
+# Declarative definitions for cluster scoped resources
+
+- Status: accepted
+- Deciders: 4n4nd, anishasthana, HumairAK, tumido
+- Date: 2021-01-18
+
+Technical Story: [issue-1](https://github.com/operate-first/odh-moc-support/issues/39), [issue-2](https://github.com/open-infrastructure-labs/ops-issues/issues/2), [pr-2](https://github.com/operate-first/apps/pull/104)
+
+## Context and Problem Statement
+
+A full Git-Ops driven approach we decided to take in the previous ADRs, makes us face additional challenges when deploying cluster scoped resources. Resources like `Namespace`, `Group` etc. should be also defined declaratively. They should be deployable via ArgoCD and allowing users to use optional encryption layer for sensitive properties/objects that may be considered a PII risk (see [`ADR 0008`](./)). Additionally the resulting layout should be flexible enough that migrating a project between clusters and/or extending the project to a multiple clusters is simple enough.
+
+## Decision Drivers
+
+- Declarative manifests in git repo.
+- Deployable via ArgoCD with cluster admin permissions.
+- Common layout that can be adopted by various downstreams and target cluster deployments.
+- Provides consistent secret management.
+- Simplifies application onboarding to ArgoCD driven deployment as a single PR.
+- Can support cluster scoped resources of various kind as well as namespaced scoped resources that are sensitive and tied to the deployed cluster scoped - e.g. a `Namespace` resource and `admin` role assignment via a `RoleBiniding`s.
```

Comment:
I _think_ I understand what you're trying to say but I'm still not sure. Are you saying the following: 

> We should be able to deploy cluster scoped resources along with any supplementary namespace scoped resources, with the added ability to encrypt these resources as necessary.

Reply:
I think we should rephrase this point, I'm certainly very confused.

Reply:
Rephrased:
> Allows to collocate cluster scoped resources along with any supplementary namespace scoped resources.

The encryption stuff was added to the 4th point as:

> Allows to collocate cluster scoped resources along with any supplementary namespace scoped resources.